src/styles/Section.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+export const Section = styled.section`
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  align-items: center;
+  justify-content: center;
+  border-radius: 1.5rem;
+  width: fit-content;
+  height: auto;
+  padding: 2rem;
+  .tweeterIcon {
```

Comment:
```suggestion
  .twitterIcon {
```
nit 😄 idk if this is intentional? if it's intentional then dont mind me haha

Reply:
thanks for pointing that out =P